### PR TITLE
CVS-57707 Add scaling option to image transformation example custom node

### PR DESCRIPTION
**Pull request by dkalinowski**

### CODE REVIEW

File: src/custom_nodes/image_transformation/README.md [hunk 1]
```diff
@@ -38,4 +39,7 @@ It will compile the library inside a docker container and save the results in `l
...
 | original_image_layout  | Input image layout. This is required to determine image shape from input shape | | &check; |
 | target_image_layout  | Output image layout. If specified and differs from original_image_layout, layout conversion will be performed | | |
+| scale  | All values will be divided by this value. When `scale_values` is specified, this value is ignored. [read more](https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model_General.html) | | |
+| scale_values  | Scale values to be used for the input image per channel. Input data will be divided those values. Values should be provided in the same order as output image color order. [read more](https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model_General.html) | | |
+| scale_values  | Mean values to be used for the input image per channel. Values will be substracted from each input image data value. Values should be provided in the same order as output image color order. [read more](https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model_General.html) | | |
```
Comment (atobiszei) [inline src/custom_nodes/image_transformation/README.md:44]:
```suggestion
| mean_values  | Mean values to be used for the input image per channel. Values will be substracted from each input image data value. Values should be provided in the same order as output image color order. [read more](https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model_General.html) | | |
```

File: src/custom_nodes/image_transformation/image_transformation.cpp [hunk 2]
```diff
@@ -31,10 +31,10 @@ int execute(const struct CustomNodeTensor* inputs, int inputsCount, struct Custo
     // If not specified (-1), the image will not be resized.
     // When specified, cv::resize is used to resize an image.
     // Original image size must not specified, input size is dynamic.
-    int targetImageHeight = get_int_parameter("target_image_height", params, paramsCount, -1);
-    int targetImageWidth = get_int_parameter("target_image_width", params, paramsCount, -1);
-    NODE_ASSERT(targetImageHeight > 0 || targetImageHeight == -1, "target image height - when specified, must be larger than 0");
-    NODE_ASSERT(targetImageWidth > 0 || targetImageWidth == -1, "target image width - when specified, must be larger than 0");
+    int _targetImageHeight = get_int_parameter("target_image_height", params, paramsCount, -1);
```
Comment (atobiszei) [inline src/custom_nodes/image_transformation/image_transformation.cpp:34]: Why this change?

File: src/custom_nodes/image_transformation/README.md [hunk 3]
```diff
@@ -38,4 +39,7 @@ It will compile the library inside a docker container and save the results in `l
 | target_image_color_order  | Output image color order. If specified and differs from original_image_color_order, color order conversion will be performed | `BGR` |  |
 | original_image_layout  | Input image layout. This is required to determine image shape from input shape | | &check; |
 | target_image_layout  | Output image layout. If specified and differs from original_image_layout, layout conversion will be performed | | |
+| scale  | All values will be divided by this value. When `scale_values` is specified, this value is ignored. [read more](https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model_General.html) | | |
+| scale_values  | Scale values to be used for the input image per channel. Input data will be divided those values. Values should be provided in the same order as output image color order. [read more](https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model_General.html) | | |
```
Comment (atobiszei) [inline src/custom_nodes/image_transformation/README.md:43]: >Input data will be divided those values.

Should be:
Input data will be divided by those values.?

File: src/custom_nodes/image_transformation/image_transformation.cpp [hunk 4]
```diff
@@ -97,18 +119,29 @@ int execute(const struct CustomNodeTensor* inputs, int inputsCount, struct Custo
         NODE_ASSERT(originalImageColorChannels == 3, "for color order BGR/RGB color channels must be equal to 3");
     }
 
-    targetImageHeight = targetImageHeight == -1 ? originalImageHeight : targetImageHeight;
-    targetImageWidth = targetImageWidth == -1 ? originalImageWidth : targetImageWidth;
+    uint64_t targetImageHeight = _targetImageHeight == -1 ? originalImageHeight : _targetImageHeight;
```
Comment (atobiszei) [inline src/custom_nodes/image_transformation/image_transformation.cpp:122]: Minor:
I think in this form this would be more readable
uint64_t targetImageHeight = _targetImageHeight != -1 ?  _targetImageHeight : originalImageHeight;

Comment (dkalinowski) [reply src/custom_nodes/image_transformation/image_transformation.cpp:122]: This is similar convention to other parameters default setting

**Comment by dkalinowski** [comment]
Moved to https://github.com/openvinotoolkit/model_server/pull/749

File: src/custom_nodes/image_transformation/README.md (hunk 3 above)
Comment (dkalinowski) [reply src/custom_nodes/image_transformation/README.md]: done
